packages/itwin/infrastructure-iot/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "@itwin/infrastructure-iot-react",
+  "version": "0.1.0",
+  "description": "This package package provides pre-built UI components and data services to integrate Infrastructure IoT real-time sensor data into an iTwin application",
+  "license": "MIT",
+  "keywords": [
+    "Bentley",
+    "Infrastructure IoT",
+    "iTwin IoT Data",
+    "Digital Twin",
+    "iTwin.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/itwin/viewer-components-react/tree/master/packages/itwin/infrastructure-iot"
+  },
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/infrastructure-iot-react.js",
+  "module": "lib/esm/infrastructure-iot-react.js",
+  "typings": "lib/cjs/infrastructure-iot-react",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "copy:assets": "npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "cover": "",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "",
+    "test": "",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "rebuild": "npm run clean && npm run build"
+  },
+  "dependencies": {
+    "@itwin/itwinui-react": "^1.34.2",
+    "axios": "^0.26.1",
+    "class-transformer": "^0.5.1",
+    "highcharts": "^10.0.0",
+    "highcharts-react-official": "^3.1.0",
+    "lodash": "^4.17.21",
+    "react-infinite-scroll-component": "^6.1.0",
+    "react-resize-detector": "^7.0.0",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.5.5"
+  },
+  "devDependencies": {
+    "@itwin/appui-abstract": "^3.1.0",
+    "@itwin/appui-layout-react": "^3.1.0",
+    "@itwin/appui-react": "^3.1.0",
+    "@itwin/build-tools": "^3.1.0",
+    "@itwin/components-react": "^3.1.0",
+    "@itwin/core-bentley": "^3.1.0",
+    "@itwin/core-common": "^3.1.0",
+    "@itwin/core-frontend": "^3.1.0",
+    "@itwin/core-geometry": "^3.1.0",
+    "@itwin/core-i18n": "^3.1.0",
+    "@itwin/core-markup": "^3.1.0",
+    "@itwin/core-orbitgt": "^3.1.0",
+    "@itwin/core-quantity": "^3.1.0",
+    "@itwin/core-react": "^3.1.0",
+    "@itwin/eslint-plugin": "^3.1.0",
+    "@itwin/imodel-components-react": "^3.1.0",
+    "@itwin/presentation-common": "^3.1.0",
```

Comment:
not sure if you need any of these presentation deps

Reply:
Removed these